### ISSUE TITLE
(C3 Runtime) Uncaught (in promise) TypeError: Cannot read property 'Release' of Null

### ISSUE THREAD

**Issue by NetOneGames** [Scirra/Construct-3-bugs#1619]
## Problem description

As per before,  just going through these errrors one by one.
I have been unable to open my project with C3 runtime.
I have stripped out as much from my project as I can while maintiaining this error.
(The project is now totally broken and will not run in C2 runtime either) 
However the console error is maintained in the C3 runtime with the left over events in attached file.

Please note that I tested this at work today on a locked down work machine.  Which has a slightly out of date build of chrome ( Version 66.0.3359.117 (Official Build) (64-bit) ) and is running on windows 7.  At time of posting I have yet to test this error on my home windows 10 setup.

## Attach a .c3p

[KLXC3RTTEST2 (2).zip](https://github.com/Scirra/Construct-3-bugs/files/2031541/KLXC3RTTEST2.2.zip)

## Steps to reproduce

1. Run the attached project (C3 Runtime)
2. Observe chrome console error as above.
3.

## Observed result

Console error
Uncaught (in promise) TypeError: Cannot read property 'Release' of Null
at C3.Instance.Release
at C3.Runtime.Step

## Expected result

C3 runtime awesomeness.

## Affected browsers

* Chrome: yes only tested

## System details

C3 Runtime R101.2 beta
Chrome ( Version 66.0.3359.117 (Official Build) (64-bit) )
Windows 7 Enterprise SP1

**Comment by Wackytoaster** [comment]
I went over it out of curiosity, this appears to be the culprit, as disabling it makes the error disappear.
![culprit](https://user-images.githubusercontent.com/31101309/40433157-a1194e30-5eab-11e8-811a-9202199c858d.PNG)

**Comment by AshleyScirra** [comment]
Thanks, should be fixed in the next release. The project just displays a black screen with the fix, but the JS error is corrected, so I assume the project just got everything deleted away?

**Comment by NetOneGames** [comment]
Yes, im deleting everything I can while trying to maintain the error to help identify what might be causing. 
definitely not expecting the file to run anything. I must have chopped out 1000+ events and files from the project. Surprised you even got a black screen with what was left!

And thanks @Wackytoaster !
